modules/claims_api/spec/lib/claims_api/power_of_attorney_pdf_constructor_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 require 'claims_api/power_of_attorney_pdf_constructor'
-require_relative '../support/pdf_matcher'
+require_relative '../../support/pdf_matcher'
```

Comment:
i dont know why i had to change this... this file was suddenly unable to find this pdf_matcher support file for some reason...